### PR TITLE
First pass at benefits.md

### PR DESCRIPTION
**Pull request by eseidel**
@felangel for your review.

### CODE REVIEW

File: benefits.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+# Introduction
+Benefits are currently written US-centric as all Shorebird employees currently
+reside in the US.  We may adapt offered benefits in country/region specific
+manners as we learn to employ persons there.
+
+## Spending Company Money
+Shorebird will pay for the items you need to get your job done.  We will provide
+you equipment when you start and you will also be provided a company card
+through which to order things needed to advanced the company.  In general the
```
Comment (felangel) [inline benefits.md:12]:
```suggestion
through which to order things needed to advance the company.  In general the
```

**Review by felangel — APPROVED** [review]
LGTM 🚀

**Comment by eseidel** [comment]
Hmm, I was supposed to accept your suggestions and failed to. 😢

**Comment by felangel** [comment]
> Hmm, I was supposed to accept your suggestions and failed to. 😢

That’s okay it was just a tiny nit. I can open a PR to fix in a few mins.

**Comment by eseidel** [comment]
I think I managed to fix it after the fact. 🤞

**Pull request merged**